recipe/meta.yaml

```diff
@@ -18,7 +18,7 @@ source:
   sha256: 80593019abbd1aec121d2048ec9e6ffa0adc3a7eeb5f6451ceb593af54ac2c03  # [win]
 
 build:
-  number: 0
+  number: 1
   skip: true  # [osx or aarch64]
```

Comment:
Currently `linux_aarch64` is skipped. So this only affects `linux_ppc64le`

That said, the Arch Migrator PR ( https://github.com/conda-forge/cuda-nvvp-feedstock/pull/1 ) configured both Linux architectures for builds. So it makes sense to configure both for cross-compilation

That way it is easy to benefit from the cross-compilation changes should `linux_aarch64` be added here